common/key/keys.go

```diff
@@ -341,31 +333,19 @@ func (s *Share) FromTOML(i interface{}) error {
 	}
 	sch, err := crypto.SchemeFromName(t.SchemeName)
 	if err != nil {
-		// we don't handle this error as it is a migration path for unspecified share's scheme
-		sch = nil
-	}
-
-	if sch != nil && s.Scheme != nil && sch.Name != s.Scheme.Name {
```

Comment:
This still seems like a worthwhile error to check

Reply:
All errors are checked. I suspect git diff chunking is making it harder to see.

Reply:
This was a migration path from when the toml files didn't have the scheme name in them. Nowadays we'd expect all Toml files to be well formed and so I'm going to get rid of all such migration paths.